### ISSUE TITLE
Remove unused maps

### ISSUE THREAD

**Issue by kerizane** [Tuxemon/Tuxemon#1257]
There are some maps that aren't used as part of the current storyline, and are probably abandoned work. This means unnecessary maintenance work sometimes takes place on these maps.

Maybe we should delete the maps or move them to a different folder.

@JaskRendix please paste in the files names you think aren't used.

**Comment by JaskRendix** [comment]
You anticipated me! Here:


- 37707_tower.tmx (no current storyline)
- 37707_town_missing.tmx (no current storyline)
- 37707_town.tmx (no current storyline)
- bedroom_test.tmx (no current storyline)
- city1_mart_junkyard.tmx (connected to the old tuxe_mart.tmx)
- gadget_store1.tmx (unaccessible from taba_town.tmx)
- gaming_hall.tmx (no current storyline)
- rubberduck_cave_01.tmx (no current storyline)
- rubberduck_city_01.tmx (no current storyline)
- rubberduck_route_01.tmx (no current storyline)
- test_pathfinding_town.tmx
- test_pathfinding.tmx
- test-tileprops.tmx
- test.tmx
- testmap.tmx
- tt_forest_route1.tmx (no current storyline)
- tt_paper_town.tmx (no current storyline)
- tt_searoute1.tmx (no current storyline)
- tt_searoute1.tmx
- tt_testing_2.tmx
- tt_testing.tmx
- tutorial.tmx

No one (Xero or Spyder) can access these maps.

**Comment by kerizane** [comment]
I think moving the unused ones into a new folder should be safe.

**Comment by JaskRendix** [comment]
Subfolder like map/unused?

**Comment by kerizane** [comment]
A subfolder within the current map folder called unused should be fine. @bitcraft ?

**Comment by JaskRendix** [comment]
Yeah bitcraft, what to do? Because I looked into the code, and I'm not sure anymore about the subfolder. I was considering the amount of work if we decide in the future to create the maps_spyder/maps_xero folder too.

Maybe inside mods/tuxemon as maps_unused? I don't know. I'm sharing some thoughts. Especially **tuxemon/states/start/__init__.py** as well as **tuxemon/graphics.py** (starting new) and the savegame load if someone saved the file. 

Update:
I was doing some testing on the parallel folder (sub of mods/tuxemon like maps) and it looks like we avoid the graphics.py issues. I was already considering a strategy on how to deal with: main.py, teleport.py, load_menu.py, __init__.py and worldstate.py

**Comment by kerizane** [comment]
With the unused maps I'm not concerned about making sure the save file still works or the maps can be navigated between. If we want to use those maps again we can just move them back into the main folder.

Bitcraft may disagree.

**Comment by DavidLiang1129** [comment]
@kerizane Excuse me, has this problem been solved yet?

**Comment by tigercoding56** [comment]
or we could make a museum of maps , that you get as a end-game reward , tbh the thing i liked the most about tuxemon was just exploring around

**Comment by bitcraft** [comment]
> A subfolder within the current map folder called unused should be fine. @bitcraft ?

sounds good to me.  we have a similar situation with art assets where there are some that are related to the game or stories, but not used.
